contracts/ROSCA.sol

```diff
@@ -275,7 +275,7 @@ contract ROSCA {
     }
     uint256 currentRoundTotalDiscounts = removeFees(potSize() - lowestBid);
     uint256 roundDiscount = currentRoundTotalDiscounts / membersAddresses.length;
```

Comment:
Why do we need to know `roundDiscount` and log it? Why not:

```
LogRoundFundsReleased(winnerAddress, lowestBid, currentRoundTotalDiscounts, currentRound);
```

Reply:
its not a `need`, more like a `want`. It is helpful to log out more information and it would also take less computation on client side to show roundDiscount.

Reply:
I think it's better to remove the roundDiscount. We don't need to save computation on client side by putting it on a smart contract.
Please take care of non-backward compatible change in the client

Reply:
actually, there's risk of non-backward compatible change. I am fine with leaving as is now. @Kivutar @knd let me know if you think we have enough time to test on the client changes?

Reply:
i think we have to remove it, because that calculation is no longer accurate, consider this scenario 
1/3 (round 1 discount) + 1/3 (round 2 discount) + 1/3 (round 3 discount) != (3/3)
0.999999 != 1